app/views/content/blog/5-reasons-to-attend-a-train-to-teach-event.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 5 reasons to attend a Train to Teach event
+date: "2021-08-26"
+author: Get Into Teaching
+images:
+  train_to_teach:
+    path: "media/images/content/blog/train-to-teach.jpg"
+    alt: "A photograph of a busy event with banners and attendees having conversations with advisers"
+description: |-
+  Whether you’re ready to apply or it’s just an idea, here are five reasons why a Train to Teach event can help you on your journey to the classroom.
+    
+keywords:
+  - train to teach events
+  - becoming a teacher
+  - applications
+  - teacher training
+tags:
+  - becoming a teacher
+  - teacher training advisers
+  - train to teach events
+  - applications
+---
+If you’re interested in training to teach but not sure on your next steps, then our popular Train to Teach events are for you. Whether you’re ready to apply or it’s just an idea, here are five reasons why a Train to Teach event can help you on your journey to the classroom.
+
+## 1. Save some time
+A Train to Teach event offers you a one stop shop of information and support.  There are a number of different zones you can visit where our expert advisers will help you understand the qualifications you need and the range of different ways you can train.  
+
+Whether you’ve recently graduated or you’re looking to change your career, in as little as two hours you will be able to gain clarity and confidence in your steps to becoming a teacher.
+
+## 2. Get answers tailored to your circumstances
+We understand that everyone’s journey into the classroom is different and your questions may be unique to you and your circumstances. You may have questions about your eligibility or a query about the extra financial support available if you’re a parent or a carer. 
+
+At a Train to Teach event we can answer all your questions and maybe some you didn’t know you had!  
+
+## 3. Make connections
+A Train to Teach event is a great way of making new connections.   You can meet like-minded individuals on their teaching journey and stay in touch with each other through one of our Facebook groups – just search for the 'Aspiring Teacher Forum' or 'Career Change to Teaching' group. 
```

Comment:
Is it worth us linking to the groups directly rather than telling people what to search for?

```suggestion
A Train to Teach event is a great way of making new connections. You can meet like-minded individuals on their teaching journey and stay in touch with each other through our [Aspiring Teacher Forum](https://www.facebook.com/groups/1357146377672255) or [Career Change to Teaching](https://www.facebook.com/groups/CareerChangetoTeaching) Facebook groups. 
```